scriptRunner/config.js

```diff
@@ -0,0 +1,5 @@
+const secrets = require('../api/secrets')
+
+module.exports = {
+    authenticationToken: secrets.read('AUTHENTICATION_TOKEN') || process.env.AUTHENTICATION_TOKEN
```

Comment:
it would be better to call this SCRIPTRUNNER_AUTHENTICATION_TOKEN no? To be able to tell things apart

Reply:
ah, no, because it is another subrepo. Oh now I saw. Ignore my comment